politeiawww/cmd/shared/shared.go

```diff
@@ -149,3 +149,47 @@ func SetConfig(config *Config) {
 func SetClient(c *Client) {
 	client = c
 }
+
+// VerifyProposal verifies a proposal's merkle root, author signature, and
+// censorship record.
+func VerifyProposal(p v1.ProposalRecord, serverPubKey string) error {
```

Comment:
We are going to need unit tests for this function where we test all failure modes.

Reply:
This was an existing unexported func in piwww.  Where do you want these tests?  I'm not seeing any in piwww.